### PR TITLE
CEDS-1862 More tests

### PR DESCRIPTION
**Pull request by PatrykRudnicki**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -28,7 +28,7 @@ lazy val scoverageSettings: Seq[Setting[_]] = Seq(
     ".*(BuildInfo|Routes|Options|TestingUtilitiesController).*",
     "logger.*\\(.*\\)"
   ).mkString(";"),
-  coverageMinimum := 70,
+  coverageMinimum := 75,
```
Comment (SteveSugden) [inline build.sbt]: :+1

File: test/controllers/consolidations/AssociateUCRSummaryControllerSpec.scala [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+    "return 200 (OK)" when {
+
+      "displayPage is invoked with mucr and ucr in cache" in {
+
+        val cachedData = AssociateUcrAnswers(mucrOptions = Some(MucrOptions("123")), associateUcr = Some(AssociateUcr(AssociateKind.Ducr, "123")))
+
+        val result = controller(cachedData).displayPage()(getRequest)
+
+        status(result) mustBe OK
```
Comment (SteveSugden) [inline test/controllers/consolidations/AssociateUCRSummaryControllerSpec.scala]: You can verify the correct page is displayed here - `verify(summaryPage).apply(any())(any(), any())`

File: test/controllers/consolidations/AssociateUCRSummaryControllerSpec.scala [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+      "submission finished with success" in {
+
+        when(submissionService.submit(any(), any[AssociateUcrAnswers])(any()))
+          .thenReturn(Future.successful((): Unit))
+
+        val cachedData = AssociateUcrAnswers(mucrOptions = Some(MucrOptions("123")), associateUcr = Some(AssociateUcr(AssociateKind.Ducr, "123")))
+
+        val result = controller(cachedData).submit()(postRequest)
+
+        status(result) mustBe OK
```
Comment (SteveSugden) [inline test/controllers/consolidations/AssociateUCRSummaryControllerSpec.scala]: ...and here with the confirmation page

File: test/controllers/consolidations/AssociateUCRSummaryControllerSpec.scala (hunk 2 above)
Comment (PatrykRudnicki) [reply test/controllers/consolidations/AssociateUCRSummaryControllerSpec.scala]: Yeah, thanks

File: test/controllers/consolidations/AssociateUCRSummaryControllerSpec.scala (hunk 3 above)
Comment (PatrykRudnicki) [reply test/controllers/consolidations/AssociateUCRSummaryControllerSpec.scala]: Thanks, fixed

**Pull request merged**